python_katas/kata_1/test.py

```diff
@@ -38,29 +43,66 @@ class TestVerbing(unittest.TestCase):
     1 Katas
     """
 
-    def test_sample(self):
-        # your code here
-        pass
+    def test_two_char(self):
+        test_word = 'mu'
+        self.assertEqual(questions.verbing(test_word), "mu")
+
+    def test_gaming(self):
+        test_word = 'gaming'
+        self.assertEqual(questions.verbing(test_word), "gamly")
+
+    def test_ing(self):
+        test_word = 'ing'
+        self.assertEqual(questions.verbing(test_word), "ly")
+
+    def test_in(self):
+        test_word = 'cartin'
+        self.assertEqual(questions.verbing(test_word), "cartining")
 
 
 class TestWordsConcatenation(unittest.TestCase):
     """
     1 Katas
     """
 
-    def test_sample(self):
-        # your code here
-        pass
+    def daft_funk(self):
```

Comment:
each function in the test case class must start with `test_`. 
In your case: `test_daft_funk`

Reply:
fixed